config/settings/prod.py

```
@@ -1,8 +1,19 @@
 from .base import *
```

Ruff (0.15.5)

[error] `from .base import *` used; unable to detect undefined names

(F403)